src/components/Documents/DocumentTable.jsx

```diff
@@ -1,17 +1,19 @@
 // React Imports
 import React, { useContext } from 'react';
 // Material UI Imports
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
 import Table from '@mui/material/Table';
 import TableBody from '@mui/material/TableBody';
 import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
-import Paper from '@mui/material/Paper';
 // Component Imports
 import { DocumentListContext } from '../../contexts';
 import { StyledTableCell } from '../Table/TableStyles';
 import DocumentTableRow from './DocumentTableRow';
-import Loading from '../Notification/Loading';
+import EmptyListNotification from '../Notification/EmptyListNotification';
```

Comment:
Think we could start moving these imports into Notification's index.js file so we can use:

`import { EmptyListNotification, LoadingAnimation } from '../Notification';`

Reply:
Completed!